network_wrangler/transit/selection.py

```diff
@@ -143,20 +143,11 @@ def validate_selection_dict(self, selection_dict: Union[dict, SelectTransitTrips
         selection_dict = selection_dict.model_dump(exclude_none=True, by_alias=True)
         WranglerLogger.debug(f"SELECT DICT - before Validation: \n{selection_dict}")
         _trip_selection_fields = list((selection_dict.get("trip_properties", {}) or {}).keys())
-        _missing_trip_fields = set(_trip_selection_fields) - set(self.net.feed.trips.columns)
-
-        if _missing_trip_fields:
-            raise TransitSelectionNetworkConsistencyError(
-                f"Fields in trip selection dictionary but not trips.txt: {_missing_trip_fields}"
-            )
+        _trip_selection_fields = list(set(_trip_selection_fields) & set(self.net.feed.trips.columns))
 
         _route_selection_fields = list((selection_dict.get("route_properties", {}) or {}).keys())
-        _missing_route_fields = set(_route_selection_fields) - set(self.net.feed.routes.columns)
+        _route_selection_fields = list(set(_route_selection_fields) & set(self.net.feed.routes.columns))
 
-        if _missing_route_fields:
-            raise TransitSelectionNetworkConsistencyError(
-                f"Fields in route selection dictionary but not routes.txt: {_missing_route_fields}"
-            )
         return selection_dict
```

Comment:
selection fields should be the overlap between trip/route columns and project properties

Reply:
The original code makes sure that all the project selection fields are IN the trip/route columns. We don't want it to fail silently if they aren't.

Reply:
I removed it because we have `shape_index` in the project properties. The `shape_index` can't be used as a selection field, it's added to project cards for the convenient of reviewing the changes. Maybe we can use warning message instead of throwing out an error?

Reply:
fixed by adding `#` before `shape_index`